samples/containers/hello-back-button/src/main/java/com/squareup/sample/hellobackbutton/HelloBackButtonWorkflow.kt

```diff
@@ -17,7 +17,7 @@ object HelloBackButtonWorkflow : StatefulWorkflow<Unit, State, Nothing, HelloBac
   enum class State : Parcelable {
     Able,
     Baker,
-    Charlie;
+    Charlie
```

Comment:
whoa - no more `;` at the end. Can we have trailing commas :)?

Reply:
This is the `standard:no-semi` rule.  I can enable it if you'd like me to. ;)

For commas... no, it's not possible.  It just doesn't work.  Lots of performance regressions and all of the internal canary branches crashed.